### PR TITLE
Add meetup list, sponsors list and footer

### PR DESCRIPTION
**Pull request by faqndo97**
## Desktop
image

## Mobile

image

Fixes: https://github.com/rubyuy/ruby.uy/issues/67

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<footer>
+  <div>
+    <h1>
+      <a href="mailto:rubyuruguay@ruby.uy">rubyuruguay@ruby.uy</a>
```
Comment (elcuervo) [inline _includes/footer.html:4]:
```suggestion
      <a href="mailto:org@ruby.uy">org@ruby.uy</a>
```

File: _includes/footer.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    </h1>
+    <h2>
+      <a href="tel:+59898455500">+598 984 555 00</a>
```
Comment (elcuervo) [inline _includes/footer.html:7]: About the 📞 . Should we have something or just remove it?

File: _includes/footer.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      <li>
+        <a href="https://github.com/rubyuy/ruby.uy"><img src="assets/images/github.svg" alt="Github Icon"></a>
+      </li>
+      <li>
+        <a href="https://www.instagram.com/ruby_uruguay/"><img src="assets/images/instagram.svg" alt="Instagram Icon"></a>
+      </li>
+    </ul>
+  </div>
+  <div>
+    <img src="assets/images/thank_you.svg" aria-hidden="true">
```
Comment (elcuervo) [inline _includes/footer.html:22]: Use absolute paths so the includes don't get 404s when used on deeply nested routes.

```suggestion
        <a href="https://twitter.com/rubymontevideo"><img src="/assets/images/twitter.svg" alt="Twitter Icon"></a>
      </li>
      <li>
        <a href="https://github.com/rubyuy/ruby.uy"><img src="/assets/images/github.svg" alt="Github Icon"></a>
      </li>
      <li>
        <a href="https://www.instagram.com/ruby_uruguay/"><img src="/assets/images/instagram.svg" alt="Instagram Icon"></a>
      </li>
    </ul>
  </div>
  <div>
    <img src="/assets/images/thank_you.svg" aria-hidden="true">
```

**Review by elcuervo — CHANGES_REQUESTED** [review]
Maravilla <3

File: _includes/footer.html (hunk 2 above)
Comment (santib) [reply _includes/footer.html:7]: 🤣 I wouldn't put my number at least. 🔪

Comment (faqndo97) [reply _includes/footer.html:7]: haha ok, I'll remove it then.

File: _includes/footer.html (hunk 3 above)
Comment (faqndo97) [reply _includes/footer.html:22]: Yes, I forgot that, thanks for caching it!

**Review by elcuervo — APPROVED** [review]
🚢 🇮🇹

**Review by santib — APPROVED** [review]

**Pull request merged**